constants.py

```diff
@@ -476,6 +485,87 @@
 kCameraServoIGain = 0.0
 kCameraServoDGain = 0.0
 
+# shooter parameters
+kTurretMotorId = 22
+kShootingMotorId = 23
+kHoodMotorId = 21
```

Comment:
Lets make `turret:21`, `hood:22`, and `flywheel:23` to keep with the general theme of indices increasing as the ball moves along the ball path